### ISSUE TITLE
Question |  macOS compatibility

### ISSUE THREAD

**Issue by DEmacRE** [cuckoosandbox/cuckoo#2191]
Hi guys

Just a general question,
Before configuring cuckoo I tried to figure out how much it is compatible to macOS and I didn't found much data on it except from the announcement on the organization site, and a bunch of issues with it...
So before setting it up I wanted to know Is cuckoo fully compatible to mac ?
Is it executing and producing full analysis on .dmg or mach-o files ?
Thanks!

**Comment by doomedraven** [comment]
you need this extension https://github.com/phdphuc/mac-a-mal-cuckoo/tree/master/analyzer/darwin/modules/packages there you can see all supported files

**Comment by seantree** [comment]
Hi @doomedraven so is the cuckoo (2.0.5) fully compatible with the latest version for mac and anyone can upload the samples through web panel??

**Comment by doomedraven** [comment]
is that the description says, i didn't test yet

**Comment by seantree** [comment]
No I dont think so, waiting for the actual working documentation. So, are you going to look into this?

**Comment by doomedraven** [comment]
yep

**Comment by seantree** [comment]
Oh that's sounds great let me know if your setup works, I am also working on this.
So here is my below problem I have setup everything I think my guest machine is not correct I am using 10.8 with host-only configuration and centos7 as the host machine so the thing is no internet on my guest machine after running iptables script which we used in for setting up the windows cuckoo sandbox.
and I used the cuckoo agent rc2. 
It would be great if someone upload the Macosx guest machine with the setup.
.
